webworker/src/test/scala/org/scalajs/macrotaskexecutor/WebWorkerMacrotaskSuite.scala

```diff
@@ -20,40 +20,37 @@ import munit.FunSuite
 import org.scalajs.dom.webworkers.Worker
 
 import scala.concurrent.Promise
-import scala.scalajs.js
-import scala.util.Try
 
 class WebWorkerMacrotaskSuite extends FunSuite {
 
   import MacrotaskExecutor.Implicits._
 
   def scalaVersion = if (BuildInfo.scalaVersion.startsWith("2"))
-    BuildInfo.scalaVersion.split("\\.").init.mkString(".")
+    BuildInfo.scalaVersion.split('.').init.mkString(".")
   else
     BuildInfo.scalaVersion
 
   def targetDir = s"${BuildInfo.baseDirectory}/target/scala-${scalaVersion}"
 
-  Try(js.isUndefined(js.Dynamic.global.window.Worker)).toOption
-    .filterNot(identity)
-    .foreach { _ =>
-      test("pass the MacrotaskSuite in a web worker") {
-        val p = Promise[Boolean]()
+  override def munitIgnore = !Set("Firefox", "Chrome").contains(BuildInfo.jsEnv)
```

Comment:
@djspiewak how about this?

Reply:
Can we put the browser check into the `buildInfoKeys` calculation instead? Basically turn the build info into something more like `"isBrowser" -> ...`. That does a better job of centralizing things so that we can add/remove browsers more easily

Reply:
Good idea. Also this is busted right now anyway lol.